### ISSUE TITLE
BUG: returning true from onCellsEdited does not stop onCellEdited being fired

### ISSUE THREAD

**Issue by ralake** [glideapps/glide-data-grid#432]
I am implementing `onCellsEdited` for bulk edits. The docs mention that if you `return true` from `onCellsEdited`, this stops `onCellEdited` from firing for those cells

```
onCellsEdited is called whenever a batch of cells is about to be edited. If the callback returns true, onCellEdited will not be called for an cells in the event.
```

However, when I `return true` from `onCellsEdited`, the `onCellEdited` function still gets called for each edit in the batch.

Version 4.2.1

**Comment by jassmith** [comment]
missed a return keyword... adding a test

**Comment by jassmith** [comment]
Fixed in 5.0.0
